docs/getting_started.md

```diff
@@ -0,0 +1,138 @@
+<cmd> docker.io/library/ubuntu`.
+- For all other images, the image name as displayed on Docker Hub, prefixed by `docker.io`.
+
+The Docker Engine client takes care of these name expansions for you so do not
+change the names you use with the Engine client or API. This is a requirement
+only when interacting with the same Docker Hub repositories through the Notary
+client.
+
+## Inspect a Docker Hub repository
+
+The most basic operation is listing the available signed tags in a repository.
+The Notary client used in isolation does not know where the trust repositories
+are located. So, you must provide the `-s` (or long form `--server`) flag to
+tell the client which repository server it should communicate with.
+
+The official Docker Hub Notary servers are located at
+`https://notary.docker.io`. Additionally, Notary stores your own signing keys,
+and a cache of previously downloaded trust metadata in a directory, provided
+with the `-d` flag. When interacting with Docker Hub repositories, you must
+instruct the client to use the associated trust directory, which by default is
+found at `.docker/trust` within the calling user's home directory (failing to
+use this directory may result in errors when publishing updates to your trust
+data):
+
+```
+$ notary -s https://notary.docker.io -d ~/.docker/trust list docker.io/library/alpine
+   NAME                                 DIGEST                                SIZE (BYTES)    ROLE
+------------------------------------------------------------------------------------------------------
+  2.6      e9cec9aec697d8b9d450edd32860ecd363f2f3174c8338beb5f809422d182c63   1374           targets
+  2.7      9f08005dff552038f0ad2f46b8e65ff3d25641747d3912e3ea8da6785046561a   1374           targets
+  3.1      e876b57b2444813cd474523b9c74aacacc238230b288a22bccece9caf2862197   1374           targets
+  3.2      4a8c62881c6237b4c1434125661cddf09434d37c6ef26bf26bfaef0b8c5e2f05   1374           targets
+  3.3      2d4f890b7eddb390285e3afea9be98a078c2acd2fb311da8c9048e3d1e4864d3   1374           targets
+  edge     878c1b1d668830f01c2b1622ebf1656e32ce830850775d26a387b2f11f541239   1374           targets
+  latest   24a36bbc059b1345b7e8be0df20f1b23caa3602e85d42fff7ecd9d0bd255de56   1377           targets
+```
+
+The output shows us the names of the tags available, the hex encoded sha256
+digest of the image manifest associated with that tag, the size of the manifest,
+and the Notary role that signed this tag into the repository. The "targets" role
+is the most common role in a simple repository. When a repository has (or
+expects) to have collaborators, you may see other "delegated" roles listed as
+signers, based on the choice of the administrator as to how they organize their
+collaborators.
+
+When you run a `docker pull` command, Docker Engine is using an integrated
+Notary library, the same one the Notary CLI uses. To request the mapping of tag
```

Comment:
The `.` here should be a ',' otherwise the following sentence `To request ...` is nonsensical. If it needs to be shortened, the addition in parentheses at the end can be turned into its own sentence: `If  you passed the \`--all\` flag, the client will use the list operation to efficiently retrieve all the mappings`